tools/move_only_function_specializations/move_only_function_specializations.py

```diff
@@ -1,7 +1,8 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-template = """template <class _Rx, class... _Types>
+def specialization(cv, ref, ref_inv, noex, noex_val, callable):
+    print(f"""template <class _Rx, class... _Types>

```

Comment:
I don't really have an objection to using f-strings instead of `format` but I would rather this function return an evaluated string, instead of directly calling print.